### PR TITLE
Added the service class method to the modelcard

### PR DESCRIPTION
**Pull request by Malikbadmus**
Related to this (comment)[https://github.com/ersilia-os/ersilia/issues/1072#issuecomment-2185852513]

**Description**

Make `ersilia catalog --local --more` verbose in its output including information about how the model was fetched.

**Changes made**

- Added `get_service_class` method to the `Localcard` class in `card.py`.
- Added `service_class` method to the `Modelcard`.
- Added a series of private method to get the status, input and output type of a model in `catalog.py`.
- Modified  Local to display this new additional information.

**Status**

- Old model local catalog output  

```JSON
[
    {
        "Identifier": "eos4wt0",
        "Slug": "morgan-fps",
        "Title": "Morgan Fingerprints"
    },
    {
        "Identifier": "eos30f3",
        "Slug": "dmpnn-herg",
        "Title": "Prediction of hERG Channel Blockers with Directed Message Passing Neural Networks"
    },
    {
        "Identifier": "eos4se9",
        "Slug": "smiles2iupac",
        "Title": "STOUT: SMILES to IUPAC name translator"
    },
    {
        "Identifier": "eos3b5e",
        "Slug": "molecular-weight",
        "Title": "Molecular weight"
    }
]
```

- New model local catalog output 

```JSON
[
    {
        "Identifier": "eos4wt0",
        "Slug": "morgan-fps",
        "Title": "Morgan Fingerprints",
        "Status": "Ready",
        "Input": "Compound",
        "Output": "Descriptor",
        "Service Class": "pulled_docker"
    },
    {
        "Identifier": "eos30f3",
        "Slug": "dmpnn-herg",
        "Title": "Prediction of hERG Channel Blockers with Directed Message Passing Neural Networks",
        "Status": "Ready",
        "Input": "Compound",
        "Output": "Score",
        "Service Class": "pulled_docker"
    },
    {
        "Identifier": "eos4se9",
        "Slug": "smiles2iupac",
        "Title": "STOUT: SMILES to IUPAC name translator",
        "Status": "Ready",
        "Input": "Compound",
        "Output": "Text",
        "Service Class": "pulled_docker"
    },
    {
        "Identifier": "eos3b5e",
        "Slug": "molecular-weight",
        "Title": "Molecular weight",
        "Status": "Ready",
        "Input": "Compound",
        "Output": "Other value",
        "Service Class": "pulled_docker"
    }
]
```

Related to #1072

### CODE REVIEW

**Comment by Malikbadmus** [comment]
@DhanshreeA , this is the output now, added three (3) additional information to it.

Result

**Comment by DhanshreeA** [comment]
> @DhanshreeA , this is the output now, added four (4) additional information to it.
> 
> Result

Not endorsing screen shots but the output looks good to me! :)

File: ersilia/hub/content/card.py [hunk 1]
```diff
@@ -714,7 +714,18 @@ def get(self, model_id):
             return card
         else:
             return None
-
+            
+    def get_service_class(self, model_id):
```
Comment (DhanshreeA) [inline ersilia/hub/content/card.py:718]: Let's add a docstring to this method!

**Review by DhanshreeA — CHANGES_REQUESTED** [review]

**Comment by DhanshreeA** [comment]
Hey @Malikbadmus great work! Thank you. Few small comments but we can absolutely merge this today.

**Comment by Malikbadmus** [comment]
> Hey @Malikbadmus great work! Thank you. Few small comments but we can absolutely merge this today.

Changes applied.

**Comment by DhanshreeA** [comment]
LGTM!

**Pull request merged**